plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/controller/CodeTransformChatController.kt

```diff
@@ -179,9 +180,12 @@ class CodeTransformChatController(
             addNewMessage(buildCompileLocalInProgressChatContent())
         }
 
+        // this should never throw the RuntimeException since invalid JDK case is already handled in previous validation step
+        val sourceJdk = ModuleUtil.findModuleForFile(moduleVirtualFile, context.project)?.tryGetJdk(context.project) ?: context.project.tryGetJdk() ?: throw RuntimeException("Unable to determine source JDK version")
+
         val selection = CustomerSelection(
             moduleVirtualFile,
-            JavaSdkVersion.JDK_1_8,
+            sourceJdk,
```

Comment:
Should not be hardcoded as JDK 8 -- need to use the actual JDK version. To get `sourceJdk`, I'm using the same exact pre-existing logic from when we had things through the dialog window (before chat invocation).

Reply:
This is the main issue being fixed ^